snet-ui/Theme/theme.ts

```diff
@@ -23,8 +23,13 @@ declare module "@mui/material/styles" {
     bgGradientHighlight?: { light?: string; main: string; dark?: string; contrastText?: string };
     bgBox?: { light?: string; main: string; dark?: string; contrastText?: string };
     bgtext?: { light?: string; main: string; dark?: string; contrastText?: string };
+    note?: { light?: string; main: string; dark?: string; contrastText?: string };
+    redtext?: { light?: string; main: string; dark?: string; contrastText?: string };
+    bgcolor?: { light?: string; main: string; dark?: string; contrastText?: string };
+    bordercolor?: { light?: string; main: string; dark?: string; contrastText?: string };
```

Comment:
why do we need these many colors?
See whether we can reuse the existing colors and new colors only if it really needed.